### ISSUE TITLE
Daily build - reporting failure to Slack fails

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-docs#975]
The "Test site and push live if we're on main branch " build appears to have been failing for months and we haven't noticed: https://github.com/medic/cht-docs/actions/workflows/daily.yml

The first failure is to do with the link checking:

```
Run ./.github/scripts/muffet.sh
  ./.github/scripts/muffet.sh
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    GOROOT: /opt/hostedtoolcache/go/1.18.10/x64
dial tcp4 127.0.0.1:1313: connect: connection refused
Error: Process completed with exit code 1.
```

The second failure is to do with reporting the failure to Slack:

```
Error sending message: Error on message: 403 Forbidden
```

Furthermore it should be renamed as it doesn't "push live" any more.

**Comment by andrablaj** [comment]
The link checking was fixed via #1013.

However, the Slack notification is still failing with:

```
Error sending message: Error on message: 403 Forbidden 
```

See [failing build](https://github.com/medic/cht-docs/actions/runs/4711220218/jobs/8355453790).

**Comment by andrablaj** [comment]
Fixed by @abbyad & @mrjones-plip 🥳
